matrix/Dcm.hpp

```diff
@@ -20,6 +20,12 @@ class Quaternion;
 template<typename Type>
 class Euler;
 
+/**
+ * Direction cosine matrix class
+ *
+ * The rotation between two coordinate frames is
```

Comment:
Something to consider is that most 3-axis gimbal use a 312 intrinsic rotation from body to sensor frame and we had to implement a 312 method  in APM when we added support for gimbal stabilisation.